tests/fx/converters/test_shape_ops_fx.py

```diff
@@ -254,3 +254,18 @@ def test_bool_shape_ops(op, kwargs):
 
     mgx_mod = convert_to_mgx(mod, [i1, i2])
     verify_outputs(mod, mgx_mod, [i1, i2])
+
+
+@pytest.mark.parametrize('size, repeat_dims', [
+    ((2, 3), (2, 4)),
+    ((1, 6), (4, 5)),
+    ((2, 1, 3), (2, 3, 1)),
+])
+def test_repeat(size, repeat_dims):
+    inp = torch.randn(size)
+    #mod_func = FuncModule(torch.repeat, *repeat_dims)
+    mod_method = MethodModule('repeat', *repeat_dims)
+
+    for mod in [mod_method]:
+        mgx_mod = convert_to_mgx(mod, [inp])
+        verify_outputs(mod, mgx_mod, inp)
```

Comment:
```suggestion
    mod_method = MethodModule('repeat', *repeat_dims)

    mgx_mod = convert_to_mgx(mod_method, [inp])
    verify_outputs(mod_method, mgx_mod, inp)
```
Seems like there is no functional call, its only implemented as a method

Reply:
There is no torch.repeat(...) functional call, so only need to support the "call_method" case.